app/javascript/app.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+  Layout
+    template(v-slot:sidebar)
+      Sidebar(v-model='drawer')
+    template(v-slot:navbar)
+      v-app-bar-nav-icon(@click='drawer = !drawer')
+      v-toolbar-title Devent
+      v-spacer
+    template(v-slot:content)
+      div
+        v-system-bar(window dark)
+          v-icon {{ titleIcon }}
+          span {{ title }}
+          v-spacer
+          v-icon mdi-refresh
+          v-icon mdi-plus-circle
+      v-container
+        router-view(@updateTitle='updateTitle')
+</template>
+
+<script>
+import Layout from './src/views/layouts/mainLayout'
+import Sidebar from './src/components/TheSidebar'
+
+export default {
+  data: function () {
+    return {
+      drawer: false,
+      title: 'Home',
```

Comment:
Somehow drawer is not getting updated when sidebar is closed. 2 way data binding not working, only in one direction..

Reply:
Moved logic to vuex store -> Fixed!